CONTRIBUTING.md

```diff
@@ -91,29 +91,58 @@ Alternatively, you can start Storybook inside of VSCode. There is a VSCode launc
 
 More information about Storybook can be found inside the **Overview** page once you have launched Storybook.
 
-### Running the unit tests and integration tests that do not require a CLI instance
+### Testing
 
-Unit tests and many integration tests do not require a copy of the CodeQL CLI.
+We have several types of tests:
 
-Outside of vscode, in the `extensions/ql-vscode` directory, run:
+* Unit tests: these live in the `tests/pure-tests/` directory
+* View tests: these live in `src/view/variant-analysis/__tests__/`
+* VSCode integration tests: these live in `src/vscode-tests/no-workspace` and `src/vscode-tests/minimal-workspace`
+* CLI integration tests: these live in `src/vscode-tests/cli-integration`
+
+The CLI integration tests require an instance of the CodeQL CLI to run so they will require some extra setup steps. When adding new tests to our test suite, please be mindful of whether they need to be in the cli-integration folder as that requires extra setup when running. If the tests don't depend on the CLI, they are better suited to being a VSCode integration test.
+
+#### Running the tests
+
+##### 1. From the terminal
+
+First move into the `extensions/ql-vscode` directory. Then, depending on which tests you want to run, use the appropriate command to run the tests:
+
+* Unit tests: `npm run test`
```

Comment:
Should this be `test:unit`?
```suggestion
* Unit tests: `npm run test:unit`
```

The plain `test` script is defined as `"test": "npm-run-all -p test:*"` so that would run all the tests and take a long time.

Reply:
Updated! Thanks!